.github/workflows/testacc.yml

```diff
@@ -92,6 +92,7 @@ jobs:
           SKIP_CLUSTER_RESOURCE_TESTS: ${{ env.SKIP_CLUSTER_RESOURCE_TESTS }}
           HOSTED_TEAM_ID: clx44rvzr01nc01o06pze6qb7
           HOSTED_USER_ID: clz3a4ymt004x01on8w5ydq8j
+          HOSTED_DUMMY_USER_ID: clzawipbm00bm01qw98vzzoca
```

Comment:
added new user id with no team or roles - using existing user causes conflicts with tests since they've already been assigned multiple roles/teams

Reply:
can we make it obvious that this user is `HOSTED_NON_TEAMS_USER_ID` and the other one is `HOSTED_HAS_TEAMS_USER_ID` or something like that